test/flow/refreshToken.test.ts

```diff
@@ -82,11 +84,11 @@ describe('flow/refreshToken.ts', () => {
 			sinon.stub(issuer, 'constructIssuer').resolves({
 				authorization_endpoint: 'https://login.salesforce.com/services/oauth2/authorize',
 				token_endpoint: 'https://login.salesforce.com/services/oauth2/token'
-			});
+			} as openidClient.Issuer);
 
 			sinon.stub(request, 'post')
 				.withArgs('https://login.salesforce.com/services/oauth2/token?grant_type=refresh_token&refresh_token=token&client_id=test&client_assertion=signed&client_assertion_type=urn%3Aietf%3Aparams%3Aoauth%3Aclient-assertion-type%3Ajwt-bearer&format=json')
-				.resolves(expectedResponse);
+				.resolves(expectedResponse as unknown as AxiosResponse);
```

Comment:
What is stopping you defining the type of expectedResponse immediately as `const: expectedResponse: AxiosResponse`? Issue with @types/sinon?

Reply:
A real AxiosResponse looks like this
```typescript
export interface AxiosResponse<T = any>  {
  data: T;
  status: number;
  statusText: string;
  headers: any;
  config: AxiosRequestConfig;
  request?: any;
}
```

Our fake looks like this:
```typescript
const expectedResponse = {
    data: expectedAccessTokenResponse
};
```

Sinon's new opinionated type checking is unhappy with this partial fake.